airflow/config_templates/airflow_local_settings.py

```diff
@@ -313,6 +313,9 @@
                 "offset_field": ELASTICSEARCH_OFFSET_FIELD,
             },
         }
+        if tuple(map(int, version.split(".")[:3])) < (2, 3, 4):
+            # todo: remove this when ES min airflow version >= 2.3.4
+            ELASTIC_REMOTE_HANDLERS["task"].update(log_id_template=ELASTICSEARCH_LOG_ID_TEMPLATE)
```

Comment:
This file is not part of the provider deployment so this code change will only be deployed in 2.5.X
I think we need to make the adjustments regarding Airflow version in ES provider code?
Did I miss something?

Reply:
sigh up too late ... yeah i think we can just simply remove log_id_template here